cylc/flow/suite_files.py

```diff
@@ -339,107 +333,6 @@ def get_contact_file(reg):
         get_suite_srv_dir(reg), SuiteFiles.Service.CONTACT)
 
 
-def get_auth_item(item, reg, owner=None, host=None, content=False):
-    """Locate/load Curve private-key/ ...etc.
-
-    Return file name, or content of file if content=True is set.
-    Files are searched from these locations in order:
-
-    1/  Server Curve ZMQ keys located in suite service directory
-        Client Curve ZMQ keys located in
-            suite service directory (private keys)
-            suite service directory/client_public_keys (public keys)
-
-    2/ For running task jobs, service directory under:
-       a/ $CYLC_SUITE_RUN_DIR for remote jobs.
-       b/ $CYLC_SUITE_RUN_DIR_ON_SUITE_HOST for local jobs or remote jobs
-          with SSH messaging.
-
-    3/ For suite on local user@host. The suite service directory.
-
-    4/ Location under $HOME/.cylc/ for remote suite control from accounts
-       that do not actually need the suite definition directory to be
-       installed:
-       $HOME/.cylc/auth/SUITE_OWNER@SUITE_HOST/SUITE_NAME/
-
-    5/ For remote suites, try locating the file from the suite service
-       directory on remote owner@host via SSH. If content=False, the value
-       of the located file will be dumped under:
-       $HOME/.cylc/auth/SUITE_OWNER@SUITE_HOST/SUITE_NAME/
-
-    """
-    if item not in [
-            SuiteFiles.Service.CONTACT,
-            SuiteFiles.Service.CONTACT2] and not isinstance(item, KeyInfo):
```

Comment:
This is a bit of special remote logic.

* On the filesystem local to the suite server we have the `CONTACT` file.
* On remote platforms we have the `CONTACT2` file.

I'm not really sure why we had this distinction. It's the same thing but it denotes that we are looking at a remote installation?

The code in `cylc/flow/task_remote_cmd.py` is still installing into `CONTACT2`. The code which reads the `CONTACT2` file has been removed in this branch. That might be a good move, I think we can probably simplify this (unless I'm forgetting about anything). If so all you need to do is to remove the `CONTACT2` logic from `cylc/flow/task_remote_cmd.py` (use `CONTACT1` instead).

Reply:
Yes, so, I questioned the necessity of `CONTACT2` in [https://github.com/cylc/cylc-flow/issues/3511](url)
Perhaps after reading @hjoliver's remarks on the discussion of the above issue; maybe best leaving it all in and shelving this work until reimplementing ssh in #3327? Or maybe not. 

> It's the same thing but it denotes that we are looking at a remote installation?

I was doing some experiments... in current Cylc 7, running a basic remote suite, setting 
`[hosts]
    [[exvcylcdev01]]
        task communication method=ssh`
in my global.rc, results in host .service dir containing... 

- `CONTACT2` file to generate and it only has one field `CYLC_COMMS_PROTOCOL_2` 

- `CONTACT`, it has 15 fields, including `CYLC_COMMS_PROTOCOL` but not `CYLC_COMMS_PROTOCOL_2`. 

Since they contain different bits of info, so two choices....
keep `CONTACT 2` as it is performing in Cylc7 or
have the field `CYLC_COMMS_PROTOCOL_2` (if it exists) get added to `CONTACT 1` and use that instead.  (perhaps there is a reason why it was not implemented like this to begin with?)

Reply:
I think kill it, git grep COMMS_PROTOCOL_2` gives only one result. @dpmatthews are we happy to kill `CONTACT_2` here?

Reply:
Killed contact2 and appended the COMMS_PROTOCOL_2 to the contact file instead.